index.md

```diff
@@ -7,4 +7,29 @@ position: 1
 tags: overview, welcome
 ---
 
-This is a placeholder document for the index page.
\ No newline at end of file
+The {{site.tp}} Knowledge Base portal gives answers to common user inquiries or issues, identified by the engineering team in the form of How-To and Troubleshooting articles. The easiest way to find relevant information is to search for key words in the *Search Knowledge Base* box at the top right corner.
+
+If you don't find information about the exact issue/scenario here, continue with the next steps in order to utilize the full potential of our support service:
+
+# **Step 1:** Browse the support resources
+The {{site.tp}} support system provides a number of **[support resources](http://docs.telerik.com/platform/appbuilder/troubleshooting/support/support-resources)** which you can browse for solutions to your problem. You can apply the suggested workarounds and continue your work without submitting a support request.
+
+# **Step 2:** Submit a support request
+
+If you cannot find a solution to your problem in the support resources, you need to submit a support request. Your current subscription plan determines the level of support assistance that you can receive. For more information about subscription plans, see <a href="http://www.telerik.com/purchase/platform" target="_blank">Buy Telerik Platform</a>.
```

Comment:
* you need to submit a support request - support **ticket** - customers can request support by submitting support tickets. 
* Buy Telerik Platform - Telerik Platform Purchasing Page - my point is to avoid the "buy" verb here so that we make sure we don't imply people have to buy something new to get support.